### ISSUE TITLE
Now CLI

### ISSUE THREAD

**Issue by MANTENN** [Urigo/graphql-modules#649]
When I import `@graphql-modules/core` I get this error:
I am using now CLI v. 16.1.2, macOS 10.14.4 (18E226)

```bash
Cannot find module '@nodelib/fs.stat'
Did you forget to add it to "dependencies" in `package.json`?
```

**Comment by MANTENN** [comment]
I am moved on with `@apollo/federation`, it's basically trying to achieve modularity with microservices.

**Comment by ecwyne** [comment]
@Urigo this happened to me as well in production on Zeit Now. Downgrading to `@graphql-modules/core@0.7.7` fixed this for me. Does `new GraphqlModule({})` now try to access the filesystem automatically?

**Comment by ecwyne** [comment]
Just tested `0.7.9` which worked. Looks like it's something that changed in `0.7.10`

**Comment by agustif** [comment]
> I am moved on with `@apollo/federation`, it's basically trying to achieve modularity with microservices.

You should probably use both, just for different use cases.


graphql-modules is better to modularize your backend, federation is great for external services/API

**Comment by ardatan** [comment]
Fixed in 0.7.11!
